pagerduty/resource_pagerduty_user.go

```diff
@@ -211,7 +212,7 @@ func resourcePagerDutyUserUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
-	return nil
+	return resourcePagerDutyUserRead(d, meta)
```

Comment:
This fixes the issue where the `html_url` was not set, even with https://github.com/terraform-providers/terraform-provider-pagerduty/pull/162/files#diff-0d158113cfaef5d534e70e6f204e0965R148

Also read after create/update is recommended by terraform team:

> Immediately after creation or updating of a resource, Terraform will have a stale state, which will result in a detected deviation on subsequent plan or applys, as Terraform refreshes its state and wants to reconcile the diff. 
> Because of this, it is **standard practice to call READ at the end of all modifications** to synchronize immediately and avoid that diff.
> ```
> func resourceExampleSimpleRead(d *schema.ResourceData, meta interface{}) error {
>    client := meta.(*ProviderApi).client
>    resource, _ := client.GetResource(d.Id())
>    d.Set("name", resource.Name)
>    d.Set("type", resource.Type)
>    return nil
> }
> 
> func resourceExampleSimpleCreate(d *schema.ResourceData, meta interface{}) error {
>    client := meta.(*ProviderApi).client
>    name := d.Get("name").(string)
>    client.CreateResource(name)
>    d.SetId(name)
>    return resourceExampleSimpleRead(d, meta)
> }
> ```
See https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#update-state-after-modification

Note: it's unusual to call update after create: https://github.com/terraform-providers/terraform-provider-pagerduty/pull/162/files#diff-0d158113cfaef5d534e70e6f204e0965R132

Reply:
Thanks for this!

Having an `Update` after `Create` is indeed unusual, but intentional because of this: https://github.com/terraform-providers/terraform-provider-pagerduty/blob/master/pagerduty/resource_pagerduty_user.go#L174. This is already on its way out though since we've deprecated that field (https://github.com/terraform-providers/terraform-provider-pagerduty/blob/master/pagerduty/resource_pagerduty_user.go#L55).